tslint.json

```diff
@@ -105,6 +105,9 @@
     "no-shadowed-variable": false,
     "object-literal-sort-keys": false,
     "array-type": [true, "generic"],
+    "no-consecutive-blank-lines": false,
+    "no-trailing-whitespace": false,
```

Comment:
Hmm....

Reply:
Found these whitespace rules to be a big distraction in my editor. I wish there was a setting to "auto fix but never warn", like `gofmt`. afaik there's no such setting-- tslint will only warn AND fix... the warnings are an attention drain, prefer to disable them.